### PR TITLE
Modernize TRavis config

### PR DESCRIPTION
**Pull request by szepeviktor**
https://github.com/szepeviktor/debian-server-tools/blob/master/webserver/Continuous-integration-Continuous-delivery.md

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,12 +3,21 @@ php:
...
+
+cache:
+  directories:
+    - $HOME/.composer/cache
+
 install:
-    composer install
+    - composer validate --strict
+    - composer install --prefer-dist --no-suggest
+    - composer normalize --dry-run
```
Comment (widoz) [inline .travis.yml]: Why --dry-run?

**Comment by widoz** [comment]
Thanks a lot for this PR, just curious about why passing `--dry-run` to `composer normalize` 
Documentation says _Show the results of normalizing, but do not modify any files_ 

If we do not want to allow file changes in this case and I guess we do not want because we want to commits those normalization why the commad is there at all?

**Comment by szepeviktor** [comment]
It is very simple

- if you change this file in a PR not conforming to composer JSON schema `--dry-run` will fail
- otherwise its exit status will be zero

https://getcomposer.org/doc/04-schema.md#json-schema

**Comment by widoz** [comment]
Thank you, PR merged.

**Comment by szepeviktor** [comment]
You're welcome.

**Pull request merged**